frontend/src/containers/Billings/BillingForm__SecondStep.tsx

```diff
@@ -0,0 +1,205 @@
+import React from "react";
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  ModalCloseButton,
+  Button,
+  VStack,
+  Box,
+  Flex,
+  Heading,
+  Checkbox,
+  Text,
+  Input,
+  Center,
+} from "@chakra-ui/react";
+import { Member } from "../../utils/fetcher";
+import {
+  Event,
+  BillingForm,
+  FormMode,
+} from "../../machines/billings/billingMachine";
+
+type Props = {
+  members: Member[];
+  formMode: FormMode;
+  billingForm: BillingForm;
+  send: (event: Event) => void;
+};
+
+const BillingFormSecondStep = ({
+  members,
+  formMode,
+  send,
+  billingForm,
+}: Props) => {
+  const handleCloseForm = () => {
+    send({ type: "CANCEL_FILL_FORM" });
+  };
+
+  const filteredMembers = members.filter(
+    (member) => member.id !== billingForm.charged_member_id
+  );
+
+  const defaultCheckedMembers = filteredMembers.map((_) => false);
+
+  const [checkedMembers, setCheckedMembers] = React.useState(
+    defaultCheckedMembers
+  );
+
+  // React.useEffect(() => {
+  //   if (formMode === FormMode.Create) {
+  // const selectedMembers = filteredMembers.map((member, index) => {
+  //   const selectedMember = checkedMembers[index] ? member.id : "";
+  //   return {
+  //     id: selectedMember,
+  //     amount: 0,
+  //   };
+  // });
+  //     send({
+  //       type: "UPDATE_FORM",
+  //       billingForm: {
+  //         ...billingForm,
+  //         members: selectedMembers,
+  //       },
+  //     });
+  //   }
+  // }, [billingForm, checkedMembers, filteredMembers, formMode, send]);
+
+  return (
+    <Modal isOpen={true} onClose={handleCloseForm} isCentered>
+      <ModalOverlay
+        bg="blackAlpha.300"
+        backdropFilter="blur(10px) hue-rotate(90deg)"
+      />
+      <ModalContent>
+        <ModalHeader textAlign="center">Billing Form</ModalHeader>
+        <ModalCloseButton />
+        <ModalBody pb={6}>
+          <Flex justifyContent="space-between">
+            <Flex>
+              <Text size="md" mr="2">
+                Total Bill
+              </Text>
+              <Text fontWeight="bold">Rp. {billingForm.bill_amount}</Text>
+            </Flex>
+            <Checkbox
+              size="lg"
+              colorScheme="cyan"
+              isChecked={billingForm?.is_bill_equally}
```

Comment:
ini si `billingForm` gak ada posibility undefined kan ya ? kayaknya gak perlu pake `?` ? jadinya tinggal `billingForm.is_bill_equally`

Reply:
This one is also solve when I update this => https://github.com/mnindrazaka/majer-otang/pull/40#discussion_r962097385